### ISSUE TITLE
configure failed when build with Android ndk

### ISSUE THREAD

**Issue by leleliu008** [rhash/RHash#179]
[config.log](https://github.com/rhash/RHash/files/7429410/config.log)

[configure-x.log](https://github.com/rhash/RHash/files/7429419/configure-x.log)

version : 1.4.2

**Comment by rhash** [comment]
Hello. Please show the output of the following two commands:
`armv7a-linux-androideabi21-clang -v`
`armv7a-linux-androideabi21-clang -V`

**Comment by leleliu008** [comment]
```
➜ aarch64-linux-android21-clang -v
Android (dev, based on r416183b) clang version 12.0.5 (https://android.googlesource.com/toolchain/llvm-project c935d99d7cf2016289302412d708641d52d2f7ee)
Target: aarch64-unknown-linux-android21
Thread model: posix
InstalledDir: /data/data/com.termux/files/home/android-ndk-r23/toolchains/llvm/prebuilt/linux-aarch64/bin
```
```
➜ aarch64-linux-android21-clang -V
clang-12: error: argument to '-V' is missing (expected 1 value)
clang-12: error: no input files
```
```
➜ aarch64-linux-android21-clang --version
Android (dev, based on r416183b) clang version 12.0.5 (https://android.googlesource.com/toolchain/llvm-project c935d99d7cf2016289302412d708641d52d2f7ee)
Target: aarch64-unknown-linux-android21
Thread model: posix
InstalledDir: /data/data/com.termux/files/home/android-ndk-r23/toolchains/llvm/prebuilt/linux-aarch64/bin
```
```
➜ aarch64-linux-android21-clang -E -dM - < /dev/null | grep clang
#define __clang__ 1
#define __clang_major__ 12
#define __clang_minor__ 0
#define __clang_patchlevel__ 5
#define __clang_version__ "12.0.5 (https://android.googlesource.com/toolchain/llvm-project c935d99d7cf2016289302412d708641d52d2f7ee)"
```

**Comment by rhash** [comment]
This bug is duplicate of the already fixed problem #169 with clang in v1.4.2.

Build rhash from the master branch or apply the fix 4dc506066cf1727b021e6352535a8bb315c3f8dc as a patch.

**Comment by leleliu008** [comment]
Thank you!
